pkg/applications/applications.go

```diff
@@ -0,0 +1,27 @@
+package applications
```

Comment:
+1 for placing shared functionality into dedicated package away from the commands. One problem we are facing is that we have too strong coupling between commands (the Cobra parts of it) and the actual business logic.

We need to get into a state where the command just deals with the options and validation of things and creates the necessary clients but then calls into other packages containing the business logic. 

Commands calling commands is another anti-patterns. Sharing of code should be via defined interfaces in the business logic interfaces. 

Anyways, just some background. Hope it makes sense.